auth/api/iam/api.go

```diff
@@ -208,85 +202,55 @@ func (r Wrapper) HandleAuthorizeRequest(ctx context.Context, request HandleAutho
 
 // GetOAuthAuthorizationServerMetadata returns the Authorization Server's metadata
 func (r Wrapper) GetOAuthAuthorizationServerMetadata(ctx context.Context, request GetOAuthAuthorizationServerMetadataRequestObject) (GetOAuthAuthorizationServerMetadataResponseObject, error) {
-	id, err := did.ParseDID(request.Did)
-	if err != nil {
-		return nil, core.InvalidInputError("authz server metadata: %w", err)
-	}
+	ownDID := idToDID(request.Id)
 
-	if id.Method != "nuts" {
-		return nil, core.InvalidInputError("authz server metadata: only did:nuts is supported")
-	}
-
-	owned, err := r.vdr.IsOwner(ctx, *id)
+	owned, err := r.vdr.IsOwner(ctx, ownDID)
 	if err != nil {
 		if didservice.IsFunctionalResolveError(err) {
 			return nil, core.NotFoundError("authz server metadata: %w", err)
 		}
-		log.Logger().WithField("did", id.String()).Errorf("authz server metadata: failed to assert ownership of did: %s", err.Error())
+		log.Logger().WithField("did", ownDID.String()).Errorf("authz server metadata: failed to assert ownership of did: %s", err.Error())
 		return nil, core.Error(500, "authz server metadata: %w", err)
 	}
 	if !owned {
 		return nil, core.NotFoundError("authz server metadata: did not owned")
 	}
 
-	identity := r.auth.PublicURL().JoinPath(apiPath, id.WithoutURL().String())
+	identity := r.auth.PublicURL().JoinPath(ownDID.WithoutURL().String())
```

Comment:
same as ClientMetadata
```suggestion
	identity := r.auth.PublicURL().JoinPath("iam", request.Id)
```